torchvision/_internally_replaced_utils.py

```diff
@@ -56,3 +56,21 @@ def _get_extension_path(lib_name):
         raise ImportError
 
     return ext_specs.origin
+
+
+def _load_library(lib_name: str) -> None:
```

Comment:
@malfet this is ok here, but given that this function is generic wrt fbcode and OSS, we could also move it to a different location (like https://github.com/pytorch/vision/blob/main/torchvision/extension.py for example) so that we don't need to re-implement it in fbcode (this file is entirely replaced in there)

otherwise, @NicolasHug we will need to implement this function in fbcode as well.

Reply:
Sorry, I've intended to remove it here